pkg/utils/otel.go

```diff
@@ -62,8 +68,12 @@ func newPropagator() propagation.TextMapPropagator {
 }
 
 func newTraceProvider(ctx context.Context, version string) (*trace.TracerProvider, error) {
+
+	grpcPort := viper.GetInt("otel_port")
+	address := fmt.Sprintf("127.0.0.1:%d", grpcPort)
```

Comment:
```suggestion
	otelGrpcPort := viper.GetInt("otel_port")
	address := fmt.Sprintf("127.0.0.1:%d", otelGrpcPort)
```
nit, we have a port for grpc service as well

Reply:
changing